rosidl_typesupport_opendds_c/CMakeLists.txt

```diff
@@ -63,10 +59,10 @@ ament_package(
   CONFIG_EXTRAS "rosidl_typesupport_opendds_c-extras.cmake.in"
 )
 
-install(
-  DIRECTORY cmake
-  DESTINATION share/${PROJECT_NAME}
-)
+#install(
+#  DIRECTORY cmake
+#  DESTINATION share/${PROJECT_NAME}
+#)
```

Comment:
```suggestion
```
Just remove it instead of commenting it out.

Reply:
Done